t/28_retries_mojoasynccaller.t

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+
+use Paws;
+use Test::Exception;
+use Test::Timer;
+use Test::More;
+
+use lib 't/lib';
+
+use Test::CustomCredentials;
+
+# Do a Volkswagen if we are in Travis. Timeout tests are very instable (since
+# when running in Travis)
+if ($ENV{IN_TRAVIS} == 1 or not defined $ENV{AUTHOR_TESTS}) {
+  ok(1, 'Travis CI detected. Skipping timeout tests');
+  done_testing;
+  exit
+}
+
+my $mojo = eval {
+  Paws->new(config => {
+    caller => 'Paws::Net::MojoAsyncCaller',
+    credentials => 'Test::CustomCredentials' 
+  });
+} or plan skip_all => "Unable to create Paws using MojoAsyncCaller";
+
+# At this point we know we have Mojo::IOLoop loaded because theabove has to
+# have loaded it
+Mojo::IOLoop->server(port => 9000, sub {
```

Comment:
It would probably be better to let the OS generate the port here, for instance by using something like the second example at https://metacpan.org/pod/Mojo::IOLoop#server

Reply:
Ahyes most likely. I had just been following the other test case examples, but that seems a better idea since it can be done.

Fixed by 82a7f7838